baseapp/baseapp.go

```diff
@@ -1099,3 +1101,18 @@ func (app *BaseApp) Close() error {
 	}
 	return app.snapshotManager.Close()
 }
+
+func (app *BaseApp) ReloadDB() error {
+	if err := app.db.Close(); err != nil {
+		return err
+	}
+	rootDir := "/root/.sei"
```

Comment:
This directory isn't always the same for all applications right? I believe we can customize the directory? Shall we pass it from somewhere instead of hard coding it?

Reply:
oh good catch I hardcoded it for testing. Updated to be config-based